djconnectwise/sync.py

```diff
@@ -501,6 +501,25 @@ def get_page(self, *args, **kwargs):
 
         return records
 
+    def create_new_note(self, target, **kwargs):
+        """
+        Send POST request to ConnectWise to create a new note and then
+        create it in the local database from the response
+        """
+
+        target_data = {}
+
+        if isinstance(target, models.Ticket):
+            target_data['id'] = target.id
+            target_data['type'] = target.record_type
+        else:
+            raise InvalidObjectException("Invalid target type for note \
```

Comment:
`InvalidObjectException` so far has been used to indicate "something is wrong with this record or related records in the database, so we can't save this", always in the context of saving info we got from ConnectWise.

This is different though- it's basically "silly developer, you can only use this for Tickets".

So let's just use `ValueError` for this, as that's a pretty typical way to indicate "silly developer, check your arguments" in Python.